WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -485,14 +485,8 @@ class PostListViewController: AbstractPostListViewController, UIViewControllerRe
     override func createPost() {
         let filterIndex = filterSettings.currentFilterIndex()
         let editor = EditPostViewController(blog: blog)
-        editor.onClose = { [weak self] changesSaved in
-            if changesSaved {
-                if let postStatus = editor.post?.status {
-                    self?.updateFilterWithPostStatus(postStatus)
-                }
-            } else {
-                self?.updateFilter(index: filterIndex)
-            }
+        editor.onClose = { [weak self] _ in
+             self?.updateFilter(index: filterIndex)
```

Comment:
```suggestion

```

It looks like we do not need to keep this at all if we also remove the `updateFilterWithPostStatus` call on line 493 which moves the tab to Draft right away. 

https://github.com/wordpress-mobile/WordPress-iOS/blob/67cc42ce581751fb52f16f710a59e41e00aa31aa/WordPress/Classes/ViewRelated/Post/PostListViewController.swift#L493